### DISCUSSION TITLE
Can't generate Admin-Token on Docker behind Traefik

### DISCUSSION THREAD

**Discussion by LucEast** [dani-garcia/vaultwarden#3768] [Q&A]
### Deployment environment

* vaultwarden version: 1.25.2

* Install method: Docker

* Clients used: -

* Reverse proxy and version: Traefik 2.10.3

* MySQL/MariaDB or PostgreSQL version:  - 

* Other relevant details: - 

### Steps to reproduce

I tried to generate a Admin-Token via `docker exec -it {ContainerName} /vaultwarden hash` and within the container itself(`docker exec -it {ContainerName} /bin/bash` `./vaultwarden hash`)

As written above my vaultwarden container sits behind a Traefik reverse-proxy. I googled the error and saw that it needs port 80 to perform this action?! However port 80 is used by my Traefik. I also tried to use the docker-compose environment variable: ` - PORT=8080` and ` - PORT=8081` but that doesn't work either. 

### Expected behaviour

Prompt for user input

### Actual behaviour

```
[INFO] No .env file found.

Error: Rocket.
[CAUSE] Bind(
    Os {
        code: 98,
        kind: AddrInUse,
        message: "Address already in use",
    },
)
```

**Comment by BlackDex** [comment]
Vaultwarden v1.25.2 is very very old and does not support the hash command. Therefore it skips it, and just tries to start Vaultwarden.
No port bindings are needed for running the hash command, so that shouldn't matter.

Just make sure you have the latest version of the Vaultwarden container.
